app/src/main/java/io/apicurio/registry/rest/RegistryExceptionMapper.java

```diff
@@ -76,6 +76,7 @@
         map.put(RuleNotFoundException.class, HTTP_NOT_FOUND);
         map.put(RuleViolationException.class, HTTP_BAD_REQUEST);
         map.put(VersionNotFoundException.class, HTTP_NOT_FOUND);
+        map.put(IllegalArgumentException.class, 422);
```

Comment:
No enum / constant for 422 ?

Reply:
There is no existing one so I've added it.